common/national_focus/ethiopia.txt

```diff
@@ -7366,7 +7366,9 @@ focus_tree = {
 		
 		cost = 10
 		available = {
-			# is_subject = no # R56
+			is_subject = no
+			is_in_faction = no
+			has_capitulated = no #To prevent the country from insantly going into a peace conference if capitulated
```

Comment:
all focuses are unavailable when capitulated by default, unless specifically specified otherwise with `vailable_if_capitulated = yes`, Ill fix this myself